server/database/connection.js

```diff
@@ -0,0 +1,11 @@
+const { Pool } = require('pg');
+require('env2')('./config.env');
+
+if (!process.env.DB_URL) throw new Error('No Database URL!!!');
+
+const options = {
+  connectionString : process.env.DB_URL,
+  ssl: true
+};
```

Comment:
change the env.DB_URL to env.DATABASE_URL to compatibility with Heroku , where the Heroku using DATABASE_URL  key 

chnage ssl to this  SSL: !connectionString.includes('localhost'),